src/QMCWaveFunctions/BsplineFactory/SplineR2R.cpp

```diff
@@ -99,6 +100,7 @@ void SplineR2R<ST>::applyRotation(const ValueMatrix& rot_mat, bool use_stored_co
   }
 
   // Apply rotation the dumb way b/c I can't get BLAS::gemm to work...
+  const auto new_coefs = new ValueType[coefs_tot_size];
```

Comment:
It would be better to avoid explicit memory management, and use a std::vector for the temporary.

Reply:
Fixed.